src/index.html

```diff
@@ -4,10 +4,47 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>Product cards by vmisai</title>
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/stars.scss">
+    <link rel="stylesheet" href="./styles/card.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="iMac A1419"
+      >
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__review-count">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-label">Price:</div>
+        <div class="card__price-total">$2,199</div>
+      </div>
```

Comment:
Use `p` tag for text according semantic 
```suggestion
        <p class="card__review-count">Reviews: 5</p>
      </div>

      <div class="card__price">
        <p class="card__price-label">Price:</p>
        <p class="card__price-total">$2,199</p>
      </div>
```